src/Microsoft.FeatureManagement/ServiceCollectionExtensions.cs

```diff
@@ -12,6 +12,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Reflection;
```

Comment:
I don't see any changes in this file related to System.Reflection, is this used?

Reply:
Good catch! I forgot to remove it.